.github/workflows/jdbc-tests.yml

```diff
@@ -40,7 +40,13 @@ jobs:
         run: |
           cd test/JDBC/
           mvn test
-      
+
+      - name: Cleanup babelfish database
+        id: cleanup
+        if: always() && steps.install-extensions.outcome == 'success'
+        run: |
+          sudo ~/postgres/bin/psql -d postgres -U runner -v user="jdbc_user" -v db="jdbc_testdb" -f .github/scripts/cleanup_babelfish_database.sql
```

Comment:
let us keep it here for jdbc tests only, for upgrade tests will add it later when top level composite action is ready

Reply:
Done!